modules/oe_content_call_proposals/config/install/core.entity_form_display.node.oe_call_proposals.default.yml

```diff
@@ -230,31 +243,44 @@ content:
         maxlength_js_truncate_html: true
     type: text_textarea
     region: content
+  oe_teaser:
+    type: text_textarea
+    weight: 16
+    region: content
+    settings:
+      rows: 5
+      placeholder: ''
+    third_party_settings:
+      maxlength:
+        maxlength_js: null
```

Comment:
We need to set the chars limit to 150.

Reply:
Yeah, let's do this, although we are not consistent on all content types with this.